aip_x2_launch/launch/pandar_node_container.launch.py

```diff
@@ -207,6 +207,27 @@ def create_parameter_dict(*args):
         extra_arguments=[{"use_intra_process_comms": LaunchConfiguration("use_intra_process")}],
     )
 
+    blockage_diag_return_component = ComposableNode(
```

Comment:
[IMO] blockage_diag_**return**_component sounds bit strange. Maybe "blockage_diag_component" is fine?

Reply:
https://github.com/tier4/aip_launcher/commit/72da1c8546f84eedf753331086c69586c40a1552